StreamChatClient_v3/ChatClient.swift

```diff
@@ -0,0 +1,85 @@
+//
+// ChatClient.swift
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+import Foundation
+
+public final class ChatClient {
+  // MARK: - Public
+
+  public let currentUser: User
```

Comment:
We use immutable `currentUser` everywhere, maybe we can just use `user` since it can't be changed?

Reply:
I forgot to mention that it's of course should be just `user`. It's really interesting why it's immutable.